Microsoft.HealthVault.Fhir.UnitTests/ToFhirTests/PersonItemToFhirTests.cs

```diff
@@ -0,0 +1,248 @@
+        {
+            const string middleName = "Mc";
+            PersonItem person = GetSamplePerson();
+            person.Name.Middle = middleName;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Name.First().Given.Any());
+            Assert.AreEqual(middleName, practitioner.Name.First().Given.First());
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenLastNameIsAddedAsFamily()
+        {
+            const string lastName = "Mc";
+            PersonItem person = GetSamplePerson();
+            person.Name.Last = lastName;
+
+            var practitioner = person.ToFhir();
+
+            Assert.AreEqual(lastName, practitioner.Name.First().Family);
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenSuffixIsAddedToSuffix()
+        {
+            const string suffix = "Jr.";
+            PersonItem person = GetSamplePerson();
+            person.Name.Suffix = new CodableValue(suffix);
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Name.First().Suffix.Any());
+            Assert.AreEqual(suffix, practitioner.Name.First().Suffix.First());
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenOrganisationIsAddedAsAnExtension()
+        {
+            const string organisation = "Interlake Hospitals";
+            PersonItem person = GetSamplePerson();
+            person.Organization = organisation;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Extension.Any(ext => ext.Url == HealthVaultExtensions.Organization));
+            Assert.AreEqual(organisation, practitioner.GetStringExtension(HealthVaultExtensions.Organization));
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenProffessionalTrainingIsAddedAsQualification()
+        {
+            const string training = "AtoZ Training";
+            PersonItem person = GetSamplePerson();
+            person.ProfessionalTraining = training;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Qualification.Any());
+            Assert.AreEqual(training, practitioner.Qualification.First().Code.Text);
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenIdIsCopiedToIdentifier()
+        {
+            const string id = "DOC3467";
+            PersonItem person = GetSamplePerson();
+            person.PersonId = id;
+
+            var practitioner = person.ToFhir();
+
+            Assert.IsTrue(practitioner.Identifier.Any());
+            Assert.AreEqual(id, practitioner.Identifier.First().Value);
+        }
+
+        [TestMethod]
+        public void WhenPersonItemTransformedToFhir_ThenContactAddressIsCopiedToAddress()
+        {
+            const string description = "14503 Apt# 102";
```

Comment:
Use Microsoft's address instead of a random one.

Reply:
These were copied from [HealthVault Docs](https://docs.microsoft.com/en-us/healthvault/reference/datatypes/medication#example) 😉